internal/actionfacts/curl_capability.go

```
@@ -0,0 +1,174 @@
+		return false
+	}
+	want := strings.ToLower(feature)
+	for _, have := range command.curlCapability.Features {
+		if have == want {
+			return true
+		}
+	}
+	return false
+}
+
+func curlCommandAllowsProtocol(command CommandFact, protocol string) bool {
```

golangci-lint (2.13.2)

[error] func curlCommandAllowsProtocol is unused

(unused)